tsconfig.base.json

```diff
@@ -12,7 +12,8 @@
     //"noUnusedLocals": true,
     "noUnusedParameters": true,
     "isolatedModules": true,
-    "checkJs": true,
+    "allowJs": true,
```

Comment:
Changes made to support the worker-wrapper JS file.

Had to change `checkJs` to `false`, otherwise `tsc` would throw errors that I didn't know how to fix 🫠:

```
packages/clis/generator/graph/find-route-worker-wrapper.js:2:26 - error TS2307: Cannot find module 'tsx/esm/api' or its corresponding type declarations.
  There are types at '/Users/icosahebron/dev/maps/node_modules/tsx/dist/esm/api/index.d.mts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.

2 import { register } from 'tsx/esm/api';
                           ~~~~~~~~~~~~~

packages/clis/generator/graph/find-route-worker-wrapper.js:30:12 - error TS7006: Parameter 'options' implicitly has an 'any' type.

30   default: options =>
              ~~~~~~~
```

For the first error: 🤷 . I still don't quite understand module resolution.

For the second error: how do you deal with such a type error in JS, where you can't add any type annotations 😳